assets/dev/js/editor/components/documents/commands/open.js

```diff
@@ -17,12 +17,20 @@ export class Open extends $e.modules.CommandBase {
 			elementor.$previewContents.find( `.elementor-${ id }` ).addClass( 'loading' );
 		}
 
+		if ( setAsInitial ) {
+			// Set the new initial document id to be sent with all requests.
```

Comment:
Why? Please explain that the next request uses the initial document id to return the preview URL that's why it has to be here and not when the document is loaded

Reply:
Good point, done